packages/components/src/KaizenProvider/index.ts

```diff
@@ -1,2 +1 @@
 export * from "./KaizenProvider"
-export * from "./ThemeProvider"
```

Comment:
We don't want this anymore?

Reply:
Nope, `KaizenProvider` replaces it and it's deprecated from `design-tokens` with suggesting `KaizenProvider` as the replacement.